source/numerics.tex

```diff
@@ -16085,7 +16085,9 @@
 \pnum
 Subclause \ref{simd} defines data-parallel types and operations on these types.
 \begin{note}
-The intent is to support acceleration through data-parallel execution resources
+SIMD stands for ``Single Instruction Stream -- Multiple Data Stream'';
```

Comment:
Maybe make this sentence the second one?

Reply:
Done; thanks! : - )